IsraelHiking.Web/src/application/components/dialogs/terms-of-service-dialog.component.ts

```diff
@@ -22,9 +22,10 @@ export class TermsOfServiceDialogComponent extends BaseMapComponent {
         super(resources);
 
         this.iAgree = false;
-        this.osmTermsOfServiceUrl = "http://wiki.osmfoundation.org/wiki/Privacy_Policy";
+        this.osmTermsOfServiceUrl = "https://wiki.osmfoundation.org/wiki/Privacy_Policy";
         this.wikimediaTermsOfServiceUrl =
             `https://wikimediafoundation.org/wiki/Terms_of_Use/${this.resources.getCurrentLanguageCodeSimplified()}`;
+        this.imgurTermsOfServiceUrl = "https://imgur.com/tos";
```

Comment:
Missing a declaration of this variable, like the other two urls.

Reply:
Added